test/Versions/TestVersionGrains2/TestVersionGrains2.csproj

```diff
@@ -70,6 +73,10 @@
       <Project>{BC1BD60C-E7D8-4452-A21C-290AEC8E2E74}</Project>
       <Name>Orleans</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\TestGrainInterfaces\TestGrainInterfaces.csproj">
+      <Project>{3972213F-189B-41D4-90FE-38D513C1106D}</Project>
+      <Name>TestGrainInterfaces</Name>
+    </ProjectReference>
```

Comment:
You'll need to add this to the vNext projects, as well as the ones with the .2017 suffix in the main tree.